tests/unit/type/set.tcl

```diff
@@ -21,6 +21,10 @@ start_server {
     test "SADD, SCARD, SISMEMBER, SMISMEMBER, SMEMBERS basics - $type" {
         create_set myset $initelems($type)
         assert_encoding $type myset
+
+        # coverage for objectComputeSize
+        assert_morethan [memory_usage myset] 0
```

Comment:
replace it with this?
https://github.com/redis/redis/blob/1190f25ca7644e5cdea0b5c8c88e07daaccc36aa/tests/unit/moduleapi/datatype2.tcl#L230

Test the equal condition about two sets.